src/clj/game/utils.clj

```diff
@@ -15,6 +15,16 @@
   [n]
   ((fnil zero? 1) n))
 
+(defn safe-inc-n
+  "Helper function to safely update a value by n. Returns a function to use with `update` / `update-in`"
+  [n]
+  #(+ (or % 0) n))
```

Comment:
Could do this: `(fnil + 0)` to be more Clojurey. Unless you are worried about things other than `nil`.

Reply:
Well, it's more whether I can still return it as a function then

Reply:
`defn safe-inc-n [n] #(+ n ((fnil identity 0) %)))`. Much more readable...
I tried to work `partial` in there for bonus points.

Reply:
Yeah, not really more readable.. I tried partial but that broke something with Atlas...